commands/health.nicole

```diff
@@ -100,7 +100,16 @@ function cmd_logs_clear {
   expect_noarg "$@"
   echo "All logs will be cleared, this event will be recorded to blank audit log."
   confirm
-  journalctl --vacuum-time=1s
+
+  rm -fr /var/lib/obmc/hostlogs/host_*.log.gz
+  cat /dev/null > /var/log/obmc-console.log
```

Comment:
Just a note. You can use `:> /var/log/obmc-console.log` for the same effect. That spares a fork-exec on `cat`.

Reply:
done